torch/_inductor/codegen/cuda/cutlass_epilogue_gen.py

```diff
@@ -0,0 +1,202 @@
+from typing import cast, List
```

Comment:
This is all new code, inspired by code within virtualized.KernelFormatterHandler. It allows to generate custom epilogue fusion functor declarations and argument initializers based of IRNodes and their inner_fn. It utilizes the V.ops overrides mechanism in order to hook into the IR.